addons/l10n_it_edi/models/account_edi_format.py

```diff
@@ -196,6 +198,39 @@ def _update_invoice_from_xml_tree(self, filename, tree, invoice):
                 return self._import_fattura_pa(tree, invoice)
         return super()._update_invoice_from_xml_tree(filename, tree, invoice)
 
+    def _decode_p7m_to_xml(self, filename, content):
+        decoded_content = remove_signature(content)
+        if not decoded_content:
+            return None
+
+        try:
+            # Some malformed XML are accepted by FatturaPA, this expends compatibility
```

Comment:
Is there any explanation / specification / ... as to what malformed xml fattura allows and why?

XML parsers are already complex beasts, but divergences in error handling and recovery is a very common source of issues.

Reply:
Unfortunately, no... It's just a common issue with XML coming from Italia and this fix was implemented by the OCA : https://github.com/OCA/l10n-italy/pull/909